grove/build/build_rpm.sh

```diff
@@ -19,6 +19,7 @@ cd "$ROOTDIR/grove"
 
 BUILDDIR="$ROOTDIR/grove/rpmbuild"
 VERSION=`cat ./VERSION`.`git rev-list --all --count`
+RELEASE=`git log -1 | awk '/^commit/ {print substr($2,1,9)}'`
```

Comment:
This doesn't get the same form as the other components, so the release id won't match.    In `trafficcontrol/build/README.md` describes how this is done for the other components -- there are functions in `trafficcontrol/build/functions.sh` that do this for you -- I recommend following the recipe in the other components to get the same result.

I do understand not wanting to have the grove main version number be different from the others (e.g. 0.1.x vs 2.3.x),  but the commit count and commit hash should be the same.